app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -57,7 +62,7 @@ if [ x$ID = xrhel ] || [ x$ID = xcentos ]; then
     yum localinstall $CONSUMER_RPM -y
     rm -f $CONSUMER_RPM
 
-    subscription-manager register --org=<%= @organization.label %> --activationkey="sample-key"
+    subscription-manager register --org=<%= @organization.label %> --activationkey="<%= @activation_key %>"
```

Comment:
I know it can be frustrating, but could you please split this to a separate PR? This one is just about deploying the SSL cert for the registration purpose. This is a valid enhancement that will be easy to test and merge.

Reply:
https://github.com/theforeman/foreman/pull/8048